synapse/util/caches/__init__.py

```diff
@@ -168,9 +205,9 @@ def register_cache(
         add_resizable_cache(cache_name, resize_callback)
 
     metric = CacheMetric(cache, cache_type, cache_name, collect_callback)
-    metric_name = "cache_%s_%s" % (cache_type, cache_name)
+    # TODO evil ??metric_name = "cache_%s_%s" % (cache_type, cache_name)
```

Comment:
What's going on here?

Reply:
bahah, just cruft. I should have checked the diff! I'll remove it :)